atlas/packages/app/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
What is this?

Reply:
This is a file generated by CRA on every run, we could possibly add it to `.gitignore`